src/main/cpp/blaze.cc

```diff
@@ -715,6 +715,20 @@ static void RunBatchMode(
 
   jvm_args_vector.insert(jvm_args_vector.end(), command_arguments.begin(),
                          command_arguments.end());
+  if (option_processor.UsingParamFile()) {
+    blaze_util::Path param_file =
+      blaze_util::Path(startup_options.output_base).GetRelative("params");
```

Comment:
I'm not sure if these param files are safe to be written to output_base, or if they need unique locations to support multiple clients using the same output_base.